frontend/src/api/AudioService.ts

```diff
@@ -0,0 +1,121 @@
+import { SpeechConfig, AutoDetectSourceLanguageConfig, AudioConfig, SpeechRecognizer, SpeechSynthesizer, ResultReason, SpeakerAudioDestination } from "microsoft-cognitiveservices-speech-sdk";
+import { AskResponse, SpeechAuth } from "./models";
+import { getSpeechAuthToken } from "./api";
+
+export class AudioService {
+    public recognizer: SpeechRecognizer | undefined;
+    private synthesiser: SpeechSynthesizer | undefined;
```

Comment:
this one is using British spelling "synthesiser", but above is "recognizer". I would recommend "synthesizer."

Reply:
done